zebra/zebra_rnh.c

```diff
@@ -302,8 +302,8 @@ static int zebra_rnh_apply_nht_rmap(int family, struct route_node *prn,
 	if (prn && re) {
 		for (nexthop = re->ng.nexthop; nexthop;
 		     nexthop = nexthop->next) {
-			ret = zebra_nht_route_map_check(rmap_family, proto,
-							&prn->p, re, nexthop);
+			ret = zebra_nht_route_map_check(
+				rmap_family, proto, &prn->p, zvrf, re, nexthop);
```

Comment:
Where does zvrf come from here?  How can this compile?  Besides the re has the re->vrf_id why do we need to pass a zvrf in here?

Reply:
On second thought I'm ok with passing in the zvrf here, but I would expect this code to at least compile

Reply:
Please check the commit 2, for this.

Reply:
individual commits being compilable is pretty important to me. please fix